Sprint-1/interpret/to-pounds.js

```diff
@@ -25,3 +25,15 @@ console.log(`£${pounds}.${pence}`);
 
 // To begin, we can start with
 // 1. const penceString = "399p": initialises a string variable with the value "399p"
+
+ // 2.const penceStringWithoutTrailingP = penceString.substring( 0, penceString.length - 1);:it initializing the string from the start(index[0]) by excluding the last character therefore,the p will be excluded//399
+
+ //3.const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0"): padStart makes sure that paddedPenceNumberString is three characters long//399
```

Comment:
In general (for any value of `penceStringWithoutTrailingP`), is `paddedPenceNumberString` exactly three characters long?